CMakeLists.txt

```diff
@@ -56,7 +56,6 @@ endif()
 include(cmake/config.cmake)
 include(cmake/gamespy.cmake)
 include(cmake/lzhl.cmake)
-include(cmake/zlib.cmake)
```

Comment:
Can this be setup in a way that it fetches zlib with cmake when vcpkg is not setup to do that instead?

Then the RTS_HAS_ZLIB thing will also not be necessary.

Or does retail game and tools not use zlib at all?

Reply:
tomsons26
> actually some maps are in zlib iirc
not available to player but left in bigs
probably a lot of the ones released with source are in zlib as well

Reply:
I do not know. Can we just leave the original zlib for VS6 in there?